didcomm_resolver/resolver.py

```diff
@@ -116,7 +118,12 @@ async def _resolve(self, profile: Profile, did: str) -> dict:
             )
             connection_ids = self._retrieve_connection_ids(records, method)
             responder = session.inject(BaseResponder)
-            assert responder
+            if not responder:
```

Comment:
The `session.inject` method has an optional parameter `required` that defaults to `True` so I don't think we'll ever hit this error path because an exception would be raised before it on line 120. I think we can safely drop this check.